### DISCUSSION TITLE
The maturity matrix is now live and ready for discussion!!!!

### DISCUSSION THREAD

**Discussion by aecurrie** [Green-Software-Foundation/green-software-maturity-matrix#15] [Announcements]
We've split things out so we can talk about each level separately.

**Comment by imix** [comment]
Hello

I have a question/proposition about one of the axis and not a particular level:

The Axis: "Minimize end user device embodied carbon waste (assume circular economy)" is defined for end user devices which are running "our" software (which is a good and necessary thing).

Wouldn't it be useuful to include the end user devices operated by the organization itself, regardless of the software it is running? Obviously there is much less control about the hardware requirements of e.g. COTS software but there still are many things that can be done to operate an end user device within the company for a longer time.

Requirements could be:

Level 1: no control over the life cycle of end user devices within the ortanisation
Level 2: defined rules for standard life-cycle replacements (e.g. every collaborator has the right to a new mobile phone every 5 years)
Level 3: device refurbishing programs are put in place (within the organisiation or outside)
Level 4: managed software life-cycle in place (e.g. we know which software version can run on which device and maybe switch devices on updates if one user needs the new version)
Level 5:  software aquisition rules for a defined lifespan of defined hardware in place

This is just a very rough draft to communicate the idea (not based on any research how this should be approached).

It could also be an option to add a new axis but making the matrix bigger is not necessarily a good thing.

thanks
Stefan
